src/models/tools/table/table-content.ts

```diff
@@ -440,11 +441,11 @@ export const TableContentModel = types
             links
       });
     },
-    removeGeometryLinks(geometryIds: string | string[], links?: ILinkProperties) {
+    removeGeometryLink(geometryId: string | string[], links?: ILinkProperties) {
```

Comment:
```suggestion
    removeGeometryLink(geometryId: string, links?: ILinkProperties) {
```
Just this last change to the types.